src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -322,6 +326,9 @@ export default {
 		NcCheckboxContent,
 	},
 
+	// We need to pass attributes to the input element
+	inheritAttrs: false,
```

Comment:
> They are as in this case it is the root element.

But it is disabled via `inheritAttrs: false` here

Reply:
For vue2 class and style are passed to the root as they are not attrs but somewhat special

Reply:
https://v3-migration.vuejs.org/breaking-changes/attrs-includes-class-style#_2-x-behavior

Reply:
> Why don't pass to the button as well?

Here I meant the attrs, not only class/style.

Reply:
Currently there is a new possibility to pass attrs to `input`, but the possibility to pass attrs to the `button` has been disabled.

Reply:
No for button type they are passed:
https://github.com/nextcloud-libraries/nextcloud-vue/pull/5507/files#diff-80b068a320cd09d28f4f8913f4c77c73754bed31f69a9b3a25353059d8573623R279

Reply:
So class & style are always set to the wrapper.
`attrs` are passed to the wrapper (the button) for button type and to the input for other types.